examples/allocation/emscripten/testdata/greet.c

```diff
@@ -0,0 +1,42 @@
+/* Note: EMSCRIPTEN_KEEPALIVE is a macro that exports the function and prevents
+   dead-code elimination on it. */
+#include <emscripten.h>
+#include <stdlib.h> // for malloc/free
+#include <string.h> // for strncat
+#include <stdint.h> // for uint64_t
+
+// _log is a WebAssembly import which prints a string (linear memory offset,
+// byteCount) to the console.
```

Comment:
maybe I would use `/* */` for all comments

Reply:
I removed the allocation example as it became really big as emscripten also requires WASI. For starters, just focused on the issue asked about.